README.md

```diff
@@ -6,6 +6,88 @@ Todo: Add table of contents
 
 # FitnessProject
 
+Getting Started:
+
+# MySQL setup for Mac 
+
+1. Download the MySQL DMG installer https://dev.mysql.com/downloads/mysql/
+2. Open the DMG file and follow the installation instructions.
+3. Open Terminal and edit your .zshrc (or .bash_profile if you're using Bash): nano ~/.zshrc
+4. Add the following line at the end of the file: export PATH="/usr/local/mysql/bin:$PATH"
+5. Save and close the file
+6. In Terminal, connect to the MySQL server: mysql -u root -p
+7. Enter your password when prompted.
+
+# MySQL setup for Windows TODO
+
+Case-insensitive table names
+
+# Local MySQL database
+
+We have migrated from PlanetScale to a local MySQL database at the moment. This section will be updated once we complete the data migration over to a new db provider.
+
+# Environment variables
+- Ask a team member for the .env variables
+
+Setup:
+1. Npm install
+2. Npm run pr - creates a pr and attaches trello card description
+- Use case: when the ticket id is in the branch name or specified as an argument to the command
+3. Npm run genapispecs - generates the specs.json
+4. Npm run resetDB - deletes the data from the db tables
+5. Npm run dbtots - generates the schema and interface for each table in the db
+
+# VSCode Setup
+
+Packages/Extensions to install when working with VSCode ->
+
+```typescript
+Linter tool: Prettier
+ESLint
+Git History
+Git Lens
+{
+    "editor.tabSize": 2,
+    "editor.indentSize": "tabSize",
+    "diffEditor.renderSideBySide": false,
+    "editor.codeActionsOnSave": {
+        "source.fixAll": "explicit",
+        "source.fixAll.eslint": "explicit",
+        "source.organizeImports": "explicit",
+        "source.addMissingImports": "explicit"
+    },
+    "javascript.updateImportsOnFileMove.enabled": "always",
+    "typescript.updateImportsOnFileMove.enabled": "always",
+    "[json]": {
+        "editor.defaultFormatter": "esbenp.prettier-vscode"
+    },
+    "[typescript]": {
+        "editor.defaultFormatter": "esbenp.prettier-vscode"
+    }
+}
+```
+
+# Environments
+
```

Comment:
Environments are these:
("devTest"),
("stagingTest"),
("staging"),
("prod")

The current environment is set using an environment variable.